src/ray/raylet/node_manager.h

```diff
@@ -454,6 +455,17 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
   void HandleDisconnectedActor(const ActorID &actor_id, bool was_local,
                                bool intentional_disconnect);
 
+  /// Handle the case where a worker is available. This should be called when a new
```

Comment:
Nit, can you rephrase the comment like this?
```
This should be called in the following cases:
1) when ...
2) when ...
```

There are too many which-s and when-s in one sentence. I'm confused ..

Reply:
will update. thanks.